src/Microsoft.IIS.WebManager/app/main/home.component.ts

```diff
@@ -24,14 +24,14 @@ import { OptionsService } from './options.service';
             <div class="sidebar" [class.nav]="_options.active">
                 <vtabs [markLocation]="true" (activate)="_options.refresh()">
                     <item [name]="'Web Sites'" [ico]="'fa fa-globe'">
-                        <dynamic [name]="'WebSiteListComponent'" [module]="'app/webserver/websites/websites.module#WebSitesModule'"></dynamic>
+                        <website-list-component></website-list-component>
```

Comment:
These objects seem to not need to be dynamic since it resolves to a static type

Reply:
There is a reason these were dynamic. Whenever a tab contains a dynamic component, the component is activated whenever the tab is selected. This allows for lazy loading of tabs. If this wasn't the case, then all the tabs in a tabs component would attempt to load their content and display at the same time. If there are a lot of tabs this can be quite heavy. I would recommend sticking with the dynamic.

[Dynamic component activate](https://github.com/Microsoft/IIS.WebManager/blob/cb66e033ccfbc3e75437e0450c33ed991dfd2a3b/src/Microsoft.IIS.WebManager/app/common/dynamic.component.ts#L40)
[Tab activate](https://github.com/Microsoft/IIS.WebManager/blob/cb66e033ccfbc3e75437e0450c33ed991dfd2a3b/src/Microsoft.IIS.WebManager/app/common/tabs.component.ts#L327)
[Tabs component calling activate](https://github.com/Microsoft/IIS.WebManager/blob/cb66e033ccfbc3e75437e0450c33ed991dfd2a3b/src/Microsoft.IIS.WebManager/app/common/tabs.component.ts#L262)